src/components/Generate.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import GenerateCorpse from './GenerateCorpse';
+import Word from './Word';
+
+import '../styles/components/generate.scss';
+
+function Generate({receiveType, typeresult1, typeresult2, favourites, addFavourites, removeFromFavourites}) {
+
+  function handleChange1(event) {
+    receiveType(event.target.value, 'type1');
```

Comment:
I wonder if `type1` and `type2` could have more descriptive names as they seem a bit vague